test3

```diff
@@ -12,23 +12,24 @@ import pathlib
 import md
 
 class TestRunner:
-    def zero_all_ram_disks(self):
+    def zero_all_disks(self):
         subprocess.check_call(["modprobe", "brd"])
         sys_blk = pathlib.Path("/sys/block")
-        for i in range(8):
-            dev = f"ram{i}"
+        for dev in self._md.devs:
             sectors = int((sys_blk / dev / "size").read_text())
             count = (sectors << 9) >> 20
             subprocess.check_call(["dd", "if=/dev/zero", f"of=/dev/{dev}",
```

Comment:
This is going to take a very long time to zero an nvme device... might be better to zero a max of 10MB now.... Looks like we're already only comparing the last 10M in the cmp command below already.

Reply:
Reviewing the code, it looks like my tests didn't actually run this (there's a bug in addition to how long it should have run for). I've fixed the of=/dev/{dev} -> of={dev} and added a new commit to address the sizing issue including superblock offsets, but is test_all not supposed to run some verification tests too?